src/main/java/core/basesyntax/FruitShopApp.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import core.basesyntax.service.converter.DataConverter;
+import core.basesyntax.service.converter.DataConverterImpl;
+import core.basesyntax.service.handler.*;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.report.ReportGenerator;
+import core.basesyntax.service.report.ReportGeneratorImpl;
+import core.basesyntax.service.shop.ShopService;
+import core.basesyntax.service.shop.ShopServiceImpl;
+import core.basesyntax.service.strategy.OperationStrategy;
+import core.basesyntax.service.strategy.OperationStrategyImpl;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.util.filereader.FileReader;
+import core.basesyntax.util.filereader.FileReaderImpl;
+import core.basesyntax.util.filewriter.FileWriter;
+import core.basesyntax.util.filewriter.FileWriterImpl;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopApp {
+    public static void main(String[] arg) {
```

Comment:
The method name 'main' should be followed by parentheses and a space, according to the method naming convention: 'main(String[] args)'.

Reply:
The definition of the main method should be public static void main(String[] args)